app/src/test/java/org/simple/clinic/bloodsugar/history/adapter/BloodSugarHistoryListItemTest.kt

```diff
@@ -136,4 +137,44 @@ class BloodSugarHistoryListItemTest {
         )
   }
 
+  @Test
+  fun `if the blood sugar type is unknown, then disallow editing`() {
```

Comment:
Thank you for adding this test. It will make it easier for us to understand the behavior this way.